test/images.js

```diff
@@ -51,66 +59,73 @@ describe('Batch Image', function() {
   describe('batch fer', function() {
     it('should get the right response format', function(done) {
 
-      if (settings.auth === false) {
+      if (settings.apiKey() === false) {
         // skip test -- indico auth keys are not available
         done();
+        return;
```

Comment:
I'm just going to go ahead and remove these checks, since the api keys are now required for any portion of the indico node package to function properly.